City-of-Helsinki/helsinki-paatokset
# PP-240: Fix time region issue with next meeting announcement

Fix issue with next meeting announcement. 

To test:
- Run `make drush-cr`
- Check Kaupunginvaltuusto page. Make sure that the announcement block displays the start time for the next Kaupunginvaltuusto meeting **in your database** correctly. (This should be the same time as in the calendar block at the bottom of the page).
- Announcement should have a link at the bottom of it that leads to stream-embed.
![image](https://user-images.githubusercontent.com/49063836/142841791-86806fdb-a95f-45c4-8620-8b218b991c96.png)
- Make sure you have Helsinki Kanava credentials set through UI. Test the migration by running `drush mim paatokset_meeting_videos --update`
- Check https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginhallitus. Stream announcements should not be displayed here and Kaupungihallitus-tag shuould have gray background.
